accel-pppd/ctrl/pppoe/cli.c

```diff
@@ -53,6 +54,11 @@ static void intf_help(char * const *fields, int fields_cnt, void *client)
 			 "pppoe interface show"
 			 " - show interfaces on which pppoe server"
 			 " started\r\n");
```

Comment:
```suggestion
			 "pppoe interface show"
			 " - show interfaces where PPPoE server is running\r\n");
```

I suppose PPPoE should be capitalized here (as a protocol name) and the message can be simpler.